eng/Versions.props

```diff
@@ -70,7 +70,7 @@
     <MicrosoftNETCoreILAsmVersion>6.0.0-rc.2.21451.6</MicrosoftNETCoreILAsmVersion>
     <MicrosoftNETSdkILVersion>6.0.0-rc.2.21451.6</MicrosoftNETSdkILVersion>
     <!-- Libraries dependencies -->
-    <SystemTextJsonVersion>6.0.0-rc.2.21451.6</SystemTextJsonVersion>
```

Comment:
@NikolaMilosavljevic nothing else in the repo seems to be referencing this package, so I bumped the version to be the latest 6.0 release.

Reply:
This is OK - thanks for updating this package version.